### PR TITLE
add e.rs

### PR DESCRIPTION
**Pull request by chenhg5**
add e.rs

### CODE REVIEW

File: e.rs [hunk 1]
```diff
@@ -0,0 +1,5 @@
+fn main() {
+	loop {
+		println!("e")
```
Comment (Br1ght0ne) [inline e.rs]: Eeeeeeeee ee eeeeeee.

**Review by Br1ght0ne — CHANGES_REQUESTED** [review]
Eee-eeeeeeee eeeeeeeeeee, eeeeee eeeeee.

File: e.rs (hunk 1 above)
Comment (chenhg5) [reply e.rs]: thanks for pointing out. I send a new pr now.

**Review by Br1ght0ne — APPROVED** [review]

**Comment by LinuxMercedes** [comment]
eeeeeeee EE eE eeEEeEeE!

**Pull request closed without merging**